install_config/install/host_preparation.adoc

```diff
@@ -710,6 +710,37 @@ An easy way to distribute your SSH keys is by using a `bash` loop:
 
 Modify the host names in the above command according to your configuration.
 
+
+[[setting-global-proxy]]
+== Setting Global Proxy Values
+
+The {product-title} installer uses the proxy settings
+in the *_/etc/environment _* file.
+
+Ensure the following domain suffixes and IP addresses are in the *_/etc/environment_* file in the `no_proxy` parameter:
+
+* Master and node host names (domain suffix).
+* Other internal host names (domain suffix).
+* Etcd IP addresses (must be IP addresses and not host names, as *etcd* access is done by IP address).
+* Docker registry IP address.
+* Kubernetes IP address, by default 172.30.0.1. Must be the value set in the
+xref:../../install_config/install/advanced_install.adoc#advanced-install-networking-variables-table[`openshift_portal_net`] parameter in the 
```

Comment:
@mburke5678 - this link only exists in enterprise 3.9 onwards. For the rest, there might be another link required. All versions where you have CPed it to(below 3.9) will require a fix.

Reply:
@vikram-redhat I missed that link. Thanks.

Reply:
@mburke5678 - 3.1 was missed. I have fixed via: https://github.com/openshift/openshift-docs/pull/9757